libgadget/forcetree.c

```diff
@@ -1020,7 +1016,7 @@ void force_treeupdate_pseudos(int no, const struct TreeBuilder tb)
  *  out just before the hydrodynamical SPH forces are computed, i.e. after
  *  density().
  */
-void force_update_hmax(int * activeset, int size)
+void force_update_hmax(int * activeset, int size, struct OctTree * tt)
```

Comment:
What is the difference between tt, tb, and tree -- which context do you use which?

Reply:
My original notation was to use tt or tb when building the tree (tt when it was a pointer to the OctTree, tb when it was an OctTree passed by value) and to use tree outside of forcetree.c when the tree was being used rather than modified. But the distinction between tt and tb got lost when I converted several internal functions to taking pointers.